config/software/datadog-logs-agent.rb

```diff
@@ -0,0 +1,14 @@
+name "datadog-logs-agent"
+always_build true
+
+build do
+  binary = "logagent"
+  logs_agent_version = "alpha"
```

Comment:
Feel free to ignore if you think it doesn't appy: 
I know you're still in alpha stage - but you might want to prepare the software definition for versioning (I presume you guys will iterate quickly so it might come in handy). You can take a look at the [datadog-process-agent](https://github.com/DataDog/dd-agent-omnibus/blob/master/config/software/datadog-process-agent.rb#L4-L13) software definition for that.

Reply:
This requires some boilerplate work to pass all parameters from jenkins to containers. I thought we could avoid this for now